ArrayLayoutRenderer was not found in '@jsonforms/angular-material'

### Describe the bug

If I try to create a custom renderer that extends the ArrayLayoutRenderer and I build the angular library I receive this error

`export 'ArrayLayoutRenderer' (imported as 'ArrayLayoutRenderer') was not found in '@jsonforms/angular-material' (possible exports: AutocompleteControlRenderer, BooleanControlRenderer, CategorizationTabLayoutRenderer, DateControlRenderer, DateControlRendererTester, GroupLayoutRenderer, HorizontalLayoutRenderer, JsonFormsAngularMaterialModule, JsonFormsDetailComponent, LabelRenderer, LabelRendererTester, LayoutChildrenRenderPropsPipe, LayoutRenderer, MasterListComponent, NumberControlRenderer, NumberControlRendererTester, ObjectControlRenderer, ObjectControlRendererTester, RangeControlRenderer, RangeControlRendererTester, TableRenderer, TableRendererTester, TextAreaRenderer, TextAreaRendererTester, TextControlRenderer, TextControlRendererTester, ToggleControlRenderer, ToggleControlRendererTester, VerticalLayoutRenderer, angularMaterialRenderers, booleanControlTester, categorizationTester, controlWithoutLabel, enumControlTester, groupLayoutTester, horizontalLayoutTester, masterDetailTester, removeSchemaKeywords, verticalLayoutTester)`


### Expected behavior

I think shoud be possible extend the Array class renderer 

### Steps to reproduce the issue

1. Create a new custom renderer
2. add import { ArrayLayoutRenderer } from '@jsonforms/angular-material/src/library/layouts/array-layout.renderer';
3. Complete the custom renderer
4. Build the angular library


### Screenshots

_No response_

### Which Version of JSON Forms are you using?

v3.2.0

### Framework

Angular

### RendererSet

Other (please specify in the Additional context field)

### Additional context

I'm creating renderers based on PrimeNg and in this specific case I need to extend the Array renderer

Hi @lucas-koehler ,
thanks for your response. 
I already tried importing from the lib folder but it does not work, i receive this error.
`Error: Cannot find module '@jsonforms/angular-material/lib/library/layouts/array-layout.renderer'`

--- a/packages/angular-material/src/library/layouts/index.ts
+++ b/packages/angular-material/src/library/layouts/index.ts
@@ -22,6 +22,7 @@
   OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
   THE SOFTWARE.
 */
+export * from './array-layout.renderer';
 export * from './horizontal-layout.renderer';
 export * from './vertical-layout.renderer';
 export * from './categorization-layout.renderer';

